File: app/src/main/java/com/semye/android/jsbridge/WebViewJavascriptBridge.java
package com.semye.android.jsbridge;

/**
 * WebView JavaScriptBridge 接口
 * https://github.com/lzyzsd/JsBridge.git
 */
public interface WebViewJavascriptBridge {

    /**
     * 向html页面发送数据
     *
     * @param data data
     */
    void send(String data);

    /**
     * 向html页面发送数据并回调
     *
     * @param data             data
     * @param responseCallback 向html页面发送数据并回调
     */
    void send(String data, CallBackFunction responseCallback);


}

File: app/src/main/java/com/semye/android/ui/webview/jsbridge/WebViewJavascriptBridge.kt
package com.semye.android.ui.webview.jsbridge

/**
 * WebView JavaScriptBridge 接口
 * https://github.com/lzyzsd/JsBridge.git
 */
interface WebViewJavascriptBridge {
    /**
     * 向html页面发送数据
     *
     * @param data data
     */
    fun send(data: String?)

    /**
     * 向html页面发送数据并回调
     *
     * @param data             data
     * @param responseCallback 向html页面发送数据并回调
     */
    fun send(data: String?, responseCallback: CallBackFunction?)
}